pkg/service/volume_service.go

```diff
@@ -71,6 +72,19 @@ func (svc *VolumeService) GetVolumeById(id string) (*nimbleos.Volume, error) {
 	return volumeResp, nil
 }
 
+// GetVolumeStats - method returns a pointer to "Stats"
+func (svc *VolumeService) GetVolumeStats(id string) (*nimbleos.Stat, error) {
```

Comment:
How might one provide filters to this call?  For example, "GET /v1/volumes/<id>/stats?sensors=reads,writes".

Reply:
Filter is not supported for now .